lms/static/sass/course/layout/_courseware_preview.scss

```diff
@@ -91,3 +91,10 @@
     margin-right: 20px;
   }
 }
+.fixed {
```

Comment:
Hmmm, the way this rule is written, anything that has a class of ``.fixed`` applied to it will inherit these rules (including anything else on a page that loads this CSS - since CSS is not scoped globally). This seems not desirable.

Also, for stateful styling, we follow the smaCSS naming convention. You can read more in our FED standards here - https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#ui-naming-conventions

With all of that said, I'd recommend using Sass's nesting here. Here's an example:

**Sass**
```scss
.wrapper-preview-menu {

    // other rules

    // STATE: Fixed to viewport (used for scrolling)
    &.is-fixed {
         // fixed-specific rules
    }
}
```